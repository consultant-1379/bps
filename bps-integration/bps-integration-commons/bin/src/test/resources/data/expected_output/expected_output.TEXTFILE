11,01/05/2009 02:42,Product1,1200,Diners,Stacy,New York,NY,United States,01/05/2009 02:23,01/05/2009 04:59,40.71417,-74.00639
12,01/05/2009 05:39,Product1,1200,Amex,Heidi,Eindhoven,Noord-Brabant,Netherlands,01/05/2009 04:55,01/05/2009 08:15,51.45,5.4666667
13,01/02/2009 09:16,Product1,1200,Mastercard,Sean,Shavano Park,TX,United States,01/02/2009 08:32,01/05/2009 09:05,29.42389,-98.49333
14,01/05/2009 10:08,Product1,1200,Visa,Georgia,Eagle,ID,United States,11/11/2008 15:53,01/05/2009 10:05,43.69556,-116.35306
15,01/02/2009 14:18,Product1,1200,Visa,Richard,Riverside,NJ,United States,12/09/2008 12:07,01/05/2009 11:01,40.03222,-74.95778
16,01/04/2009 01:05,Product1,1200,Diners,Leanne,Julianstown,Meath,Ireland,01/04/2009 00:00,01/05/2009 13:36,53.6772222,-6.3191667
17,01/05/2009 11:37,Product1,1200,Visa,Janet,Ottawa,Ontario,Canada,01/05/2009 09:35,01/05/2009 19:24,45.4166667,-75.7
18,01/06/2009 05:02,Product1,1200,Diners,barbara,Hyderabad,Andhra Pradesh,India,01/06/2009 02:41,01/06/2009 07:52,17.3833333,78.4666667
19,01/06/2009 07:45,Product2,3600,Visa,Sabine,London,England,United Kingdom,01/06/2009 07:00,01/06/2009 09:17,51.52721,0.14559
20,01/02/2009 07:35,Product1,1200,Diners,Hani,Salt Lake City,UT,United States,12/30/08 5:44,01/06/2009 10:52,40.76083,-111.89028