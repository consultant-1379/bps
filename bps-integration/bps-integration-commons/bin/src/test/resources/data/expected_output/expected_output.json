{"ID":11,"TRANSACTION_DATE":"01/05/2009 02:42","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Diners","NAME":"Stacy","CITY":"New York","STATE":"NY","COUNTRY":"United States","ACCOUNT_CREATED":"01/05/2009 02:23","LAST_LOGIN":"01/05/2009 04:59","LATITUDE":40.71417,"LONGITUDE":-74.00639}
{"ID":12,"TRANSACTION_DATE":"01/05/2009 05:39","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Amex","NAME":"Heidi","CITY":"Eindhoven","STATE":"Noord-Brabant","COUNTRY":"Netherlands","ACCOUNT_CREATED":"01/05/2009 04:55","LAST_LOGIN":"01/05/2009 08:15","LATITUDE":51.45,"LONGITUDE":5.4666667}
{"ID":13,"TRANSACTION_DATE":"01/02/2009 09:16","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Mastercard","NAME":"Sean","CITY":"Shavano Park","STATE":"TX","COUNTRY":"United States","ACCOUNT_CREATED":"01/02/2009 08:32","LAST_LOGIN":"01/05/2009 09:05","LATITUDE":29.42389,"LONGITUDE":-98.49333}
{"ID":14,"TRANSACTION_DATE":"01/05/2009 10:08","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"Georgia","CITY":"Eagle","STATE":"ID","COUNTRY":"United States","ACCOUNT_CREATED":"11/11/2008 15:53","LAST_LOGIN":"01/05/2009 10:05","LATITUDE":43.69556,"LONGITUDE":-116.35306}
{"ID":15,"TRANSACTION_DATE":"01/02/2009 14:18","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"Richard","CITY":"Riverside","STATE":"NJ","COUNTRY":"United States","ACCOUNT_CREATED":"12/09/2008 12:07","LAST_LOGIN":"01/05/2009 11:01","LATITUDE":40.03222,"LONGITUDE":-74.95778}
{"ID":16,"TRANSACTION_DATE":"01/04/2009 01:05","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Diners","NAME":"Leanne","CITY":"Julianstown","STATE":"Meath","COUNTRY":"Ireland","ACCOUNT_CREATED":"01/04/2009 00:00","LAST_LOGIN":"01/05/2009 13:36","LATITUDE":53.6772222,"LONGITUDE":-6.3191667}
{"ID":17,"TRANSACTION_DATE":"01/05/2009 11:37","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"Janet","CITY":"Ottawa","STATE":"Ontario","COUNTRY":"Canada","ACCOUNT_CREATED":"01/05/2009 09:35","LAST_LOGIN":"01/05/2009 19:24","LATITUDE":45.4166667,"LONGITUDE":-75.7}
{"ID":18,"TRANSACTION_DATE":"01/06/2009 05:02","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Diners","NAME":"barbara","CITY":"Hyderabad","STATE":"Andhra Pradesh","COUNTRY":"India","ACCOUNT_CREATED":"01/06/2009 02:41","LAST_LOGIN":"01/06/2009 07:52","LATITUDE":17.3833333,"LONGITUDE":78.4666667}
{"ID":19,"TRANSACTION_DATE":"01/06/2009 07:45","PRODUCT":"Product2","PRICE":3600,"PAYMENT_TYPE":"Visa","NAME":"Sabine","CITY":"London","STATE":"England","COUNTRY":"United Kingdom","ACCOUNT_CREATED":"01/06/2009 07:00","LAST_LOGIN":"01/06/2009 09:17","LATITUDE":51.52721,"LONGITUDE":0.14559}
{"ID":20,"TRANSACTION_DATE":"01/02/2009 07:35","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Diners","NAME":"Hani","CITY":"Salt Lake City","STATE":"UT","COUNTRY":"United States","ACCOUNT_CREATED":"12/30/08 5:44","LAST_LOGIN":"01/06/2009 10:52","LATITUDE":40.76083,"LONGITUDE":-111.89028}
{"ID":1,"TRANSACTION_DATE":"01/02/2009 06:17","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Mastercard","NAME":"carolina","CITY":"Basildon","STATE":"England","COUNTRY":"United Kingdom","ACCOUNT_CREATED":"01/02/2009 06:00","LAST_LOGIN":"01/02/2009 06:08","LATITUDE":51.5,"LONGITUDE":-1.1166667}
{"ID":2,"TRANSACTION_DATE":"01/02/2009 04:53","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"Betina","CITY":"Parkville","STATE":"MO","COUNTRY":"United States","ACCOUNT_CREATED":"01/02/2009 04:42","LAST_LOGIN":"01/02/2009 07:49","LATITUDE":39.195,"LONGITUDE":-94.68194}
{"ID":3,"TRANSACTION_DATE":"01/02/2009 13:08","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Mastercard","NAME":"Federica e Andrea","CITY":"Astoria","STATE":"OR","COUNTRY":"United States","ACCOUNT_CREATED":"01/01/2009 16:21","LAST_LOGIN":"01/03/2009 12:32","LATITUDE":46.18806,"LONGITUDE":-123.83}
{"ID":4,"TRANSACTION_DATE":"01/03/2009 14:44","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"Gouya","CITY":"Echuca","STATE":"Victoria","COUNTRY":"Australia","ACCOUNT_CREATED":"9/25/05 21:13","LAST_LOGIN":"01/03/2009 14:22","LATITUDE":-36.1333333,"LONGITUDE":144.75}
{"ID":5,"TRANSACTION_DATE":"01/04/2009 12:56","PRODUCT":"Product2","PRICE":3600,"PAYMENT_TYPE":"Visa","NAME":"Gerd W","CITY":"Cahaba Heights","STATE":"AL","COUNTRY":"United States","ACCOUNT_CREATED":"11/15/08 15:47","LAST_LOGIN":"01/04/2009 12:45","LATITUDE":33.52056,"LONGITUDE":-86.8025}
{"ID":6,"TRANSACTION_DATE":"01/04/2009 13:19","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"LAURENCE","CITY":"Mickleton","STATE":"NJ","COUNTRY":"United States","ACCOUNT_CREATED":"9/24/08 15:19","LAST_LOGIN":"01/04/2009 13:04","LATITUDE":39.79,"LONGITUDE":-75.23806}
{"ID":7,"TRANSACTION_DATE":"01/04/2009 20:11","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Mastercard","NAME":"Fleur","CITY":"Peoria","STATE":"IL","COUNTRY":"United States","ACCOUNT_CREATED":"01/03/2009 09:38","LAST_LOGIN":"01/04/2009 19:45","LATITUDE":40.69361,"LONGITUDE":-89.58889}
{"ID":8,"TRANSACTION_DATE":"01/02/2009 20:09","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Mastercard","NAME":"adam","CITY":"Martin","STATE":"TN","COUNTRY":"United States","ACCOUNT_CREATED":"01/02/2009 17:43","LAST_LOGIN":"01/04/2009 20:01","LATITUDE":36.34333,"LONGITUDE":-88.85028}
{"ID":9,"TRANSACTION_DATE":"01/04/2009 13:17","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Mastercard","NAME":"Renee Elisabeth","CITY":"Tel Aviv","STATE":"Tel Aviv","COUNTRY":"Israel","ACCOUNT_CREATED":"01/04/2009 13:03","LAST_LOGIN":"01/04/2009 22:10","LATITUDE":32.0666667,"LONGITUDE":34.7666667}
{"ID":10,"TRANSACTION_DATE":"01/04/2009 14:11","PRODUCT":"Product1","PRICE":1200,"PAYMENT_TYPE":"Visa","NAME":"Aidan","CITY":"Chatou","STATE":"Ile-de-France","COUNTRY":"France","ACCOUNT_CREATED":"06/03/2008 04:22","LAST_LOGIN":"01/05/2009 01:17","LATITUDE":48.8833333,"LONGITUDE":2.15}